acl-lib/src/cmr/acl/core.clj

```diff
@@ -140,6 +144,10 @@
   "The cache key for the token to subscription management permission cache."
   :token-smp)
 
+(def CONCEPT_MAP_CACHE_TIME
+  "The number of milliseconds token information will be cached."
```

Comment:
I initially made a separate cache time for this since I wasn't sure how long I wanted to cache for, but I ended up leaving it at the same duration as the other time. I can remove this one and use the other cache time value with the same 5-minute duration, but in the event where we want to change how long this is cached for, we'd need to separate them out again.